pxtlib/tutorialValidator.ts

```diff
@@ -107,7 +111,9 @@ namespace pxt.tutorial {
     function extractBlockSnippet(tutorial: TutorialOptions, indexdb: pxt.Map<pxt.Map<number>>) {
         const { tutorialStepInfo, tutorialStep } = tutorial;
         const body = tutorial.tutorialStepInfo[tutorialStep].hintContentMd;
+
         let hintCode = "";
+        let requiredBlocks = "";
```

Comment:
This isn't used? Fine to leave in if you were planning on adding something else in the future and it just snuck in now by accident~